### DISCUSSION TITLE
Termux not upgrading to 3.0.1

### DISCUSSION THREAD

**Discussion by ShakeBake** [lncrawl/lightnovel-crawler#1715] [Q&A] [answered]
Tried untintalling termux completely. Un-installing lncrawler and re upgrading but still stuck on 2.34.
Termux from f-droid and attempting on a Fold4.

**Comment by dipu-bd** [comment, marked as answer]
there is a issue with minify-html. it is failing to be installed in termux

follow #1713 

it'll be solved in the next version.
